ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -844,6 +849,8 @@ def get_execute_function_sql_command(function: "FunctionInfo", parameters: Dict[
                         param_value = base64.b64encode(param_value).decode("utf-8")
                     # Use ubbase64 to restore binary values.
                     arg_clause += f"unbase64('{param_value}')"
+                elif param_info.type_name == ColumnTypeName.STRING:
+                    arg_clause += f"'{param_value}'"
```

Comment:
Is this needed? Line 871 converts everything else to string :)

Reply:
lol yeah not needed at all. oof